pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -82,14 +82,28 @@ type Range struct {
 
 // NewAllocatorCIDRRange creates a Range over a net.IPNet, calling allocatorFactory to construct the backing store.
 func NewAllocatorCIDRRange(cidr *net.IPNet, allocatorFactory allocator.AllocatorFactory) (*Range, error) {
-	max := RangeSize(cidr)
-	base := bigForIP(cidr.IP)
+	max := utilnet.RangeSize(cidr)
```

Comment:
This rangesize is not capping to 16kb, max is 2^64

https://github.com/kubernetes/utils/blob/3d5a2fe318e4201932f44a9b6ee4ef0c6221f1d3/net/net.go#L168-L180

Reply:
ups, it's capped below , scratch it